### PR TITLE
Feature/issue 14 fsoi compare plots

### PR DESCRIPTION
**Pull request by hahnd**
Pull request includes two issues: issue 17 update compare_fsoi.py to accept a list of centers on the command line, while maintaining color consistency for each center; and issue 14 add inter-agency comparison plots to the web application.

### CODE REVIEW

File: scripts/compare_fsoi.py [hunk 1]
```diff
@@ -59,14 +59,15 @@ def sort_centers(DF,pref):
...
     parser = ArgumentParser(description = 'Create and Plot Comparison Observation Impact Statistics',formatter_class=ArgumentDefaultsHelpFormatter)
-    parser.add_argument('--rootdir',help='root path to directory',type=str,default='/scratch3/NCEPDEV/stmp2/Rahul.Mahajan/test/Thomas.Auligne/FSOI',required=False)
+    parser.add_argument('--rootdir',help='root path to directory',type=str,required=True)
     parser.add_argument('--platform',help='platforms to plot',type=str,default='full',choices=['full','conv','rad'],required=False)
-    parser.add_argument('--cycle',help='cycle to process',nargs='+',type=int,default=[0],choices=[0,6,12,18],required=False)
-    parser.add_argument('--norm',help='metric norm',type=str,default='dry',choices=['dry','moist'],required=False)
+    parser.add_argument('--cycle',help='cycle to process',nargs='+',type=int,choices=[0,6,12,18],required=True)
+    parser.add_argument('--norm',help='metric norm',type=str,choices=['dry','moist'],required=True)
     parser.add_argument('--savefigure',help='save figures',action='store_true',required=False)
+    parser.add_argument('--centers', help='comma-separated list of centers', type=str, required=True)
```
Comment (aerorahul) [inline scripts/compare_fsoi.py]: Instead if you add nargs='+', you won't need to do the split(',') and strip on line 79.
The input argument is then simply --centers GMAO EMC NRL.
Also, provide a choices=['GMAO', 'NRL', ...] to ensure that correct centers are provided.

Comment (hahnd) [reply scripts/compare_fsoi.py]: Thanks for the suggestion here.  I have made this change.

**Comment by aerorahul** [comment]
At some point, I recommend moving the NAVY specific scripts to a NAVY specific directory (it may or may not be under scripts).
A little re-organization would be beneficial before this directory becomes a dump of all scripts.
I leave it to you to decide how best to compartmentalize the different functionality being added.

**Comment by hahnd** [comment]
I agree.  I will give it some thought and send a proposal in the next week.

**Pull request merged**